beta-src/src/components/ui/WDMain.tsx

```diff
@@ -1,21 +1,9 @@
 import * as React from "react";
-import { fetchGameData, gameOverview } from "../../state/game/game-api-slice";
-import { useAppDispatch, useAppSelector } from "../../state/hooks";
 import WDUI from "./WDUI";
 
 const WDMapController = React.lazy(() => import("../map/WDMapController"));
 
 const WDMain: React.FC = function (): React.ReactElement {
-  const { user, gameID } = useAppSelector(gameOverview);
```

Comment:
Is this what was causing the double loading?

Reply:
Pretty much, because we're calling dispatching and then using these variables, so on new dispatch it detects a change.